tests/unit/XMLDataFormatterTest.php

```diff
@@ -74,4 +76,51 @@ public function testConvertStringToArrayMultipleEntitiesException()
         $formatter = new XMLDataFormatter();
         $formatter->convertStringToArray($inputXML);
     }
+
+    /**
+     * Data provider for trailing slash configuration
+     */
+    public function trailingSlashProvider(): array
```

Comment:
```suggestion
    public function provideConvertDataObjectWithoutHeaderClassNameAttribute(): array
```
As per https://docs.silverstripe.org/en/6/contributing/coding_conventions/php_coding_conventions/#unit-testing